src/test/configuration/index.js

```diff
@@ -3,6 +3,12 @@ const axios = require('axios');
 const { startTestServer, isStageTesting, port } = require('../../server.js');
 axios.defaults.baseURL = `http://localhost:${port}`;
 
+// Check if user is dev before running test
+if (process.env.MYSQL_USER !== 'dev') {
+  console.error('Trying to run test with user that is not dev');
+  setTimeout(() => process.exit(0));
```

Comment:
Good stuff! It's simpler than I anticipated tbh 😅 
Probably don't need to prolong it with set timeout though.

Reply:
I would remove it